<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[database connected successfully]]>
    </line>
    <line>
      <![CDATA[browser launch successfully]]>
    </line>
    <line>
      <![CDATA[logged in successfully]]>
    </line>
    <line>
      <![CDATA[logged out successfully]]>
    </line>
    <line>
      <![CDATA[logged in successfully]]>
    </line>
    <line>
      <![CDATA[take screenshot Sucessfully]]>
    </line>
    <line>
      <![CDATA[logged out successfully]]>
    </line>
    <line>
      <![CDATA[Browser close sucessfully]]>
    </line>
    <line>
      <![CDATA[Closed Database successfully]]>
    </line>
  </reporter-output>
  <suite started-at="2024-02-18T12:32:03 IST" name="Default suite" finished-at="2024-02-18T12:32:46 IST" duration-ms="42696">
    <groups>
    </groups>
    <test started-at="2024-02-18T12:32:03 IST" name="Default test" finished-at="2024-02-18T12:32:46 IST" duration-ms="42696">
      <class name="TestNG.ListenerUtilization">
        <test-method is-config="true" signature="databaseconnection()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:03 IST" name="databaseconnection" finished-at="2024-02-18T12:32:03 IST" duration-ms="8" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[database connected successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- databaseconnection -->
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:03 IST" name="launchBrowser" finished-at="2024-02-18T12:32:05 IST" duration-ms="2409" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[browser launch successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:05 IST" name="login" finished-at="2024-02-18T12:32:25 IST" duration-ms="19330" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged in successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="GetErrorScreenshot()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:25 IST" name="GetErrorScreenshot" finished-at="2024-02-18T12:32:25 IST" duration-ms="10" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at TestNG.ListenerUtilization.GetErrorScreenshot(ListenerUtilization.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetErrorScreenshot -->
        <test-method is-config="true" signature="logout()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:25 IST" name="logout" finished-at="2024-02-18T12:32:41 IST" duration-ms="15581" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged out successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="login()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:41 IST" name="login" finished-at="2024-02-18T12:32:42 IST" duration-ms="1115" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged in successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="getScreenshot()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:42 IST" name="getScreenshot" finished-at="2024-02-18T12:32:42 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at TestNG.ListenerUtilization.getScreenshot(ListenerUtilization.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[take screenshot Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- getScreenshot -->
        <test-method is-config="true" signature="logout()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:42 IST" name="logout" finished-at="2024-02-18T12:32:46 IST" duration-ms="3674" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged out successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:46 IST" name="closeBrowser" finished-at="2024-02-18T12:32:46 IST" duration-ms="165" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Browser close sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="databaseDisconnected()[pri:0, instance:TestNG.ListenerUtilization@23c30a20]" started-at="2024-02-18T12:32:46 IST" name="databaseDisconnected" finished-at="2024-02-18T12:32:46 IST" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Closed Database successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- databaseDisconnected -->
      </class> <!-- TestNG.ListenerUtilization -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
